pages/sigmaker.tsx

```diff
@@ -1,9 +1,109 @@
 import type { NextPage } from "next";
-import { ThemeProvider } from "@mui/material";
+import {
+  Button,
+  Container,
+  Divider,
+  FormControl,
+  Grid,
+  Stack,
+  TextField,
+  ThemeProvider,
+  Typography,
+} from "@mui/material";
 import { theme } from "../styles/theme";
+import { useTheme } from "@mui/material/styles";
+import { ChangeEvent, useState } from "react";
+import useMediaQuery from "@mui/material/useMediaQuery";
 
 const Sigmaker: NextPage = () => {
-  return <ThemeProvider theme={theme}></ThemeProvider>;
+  const themeMUI = useTheme();
+  const isNotMobile = useMediaQuery(themeMUI.breakpoints.up("sm"));
+
+  const [formData, setFormData] = useState({
+    fullName: "",
+    title: "",
+    phone: "",
+    email: "",
+  });
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    setFormData({
+      ...formData,
+      [name]: value,
+    });
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Container sx={{ mt: 4 }}>
+        <Typography variant="h3"> Signature Maker </Typography>
+        <Divider />
+        <br />
+        <Typography variant="h5"> Enter your info here! </Typography>
+        <FormControl sx={{ my: 3 }}>
+          <Stack spacing={3}>
+            <TextField
+              name="fullName"
+              value={formData.fullName}
+              onChange={handleChange}
+              label="Full name"
+              id="filled-size-small"
+              variant="filled"
+              size="small"
+              sx={{ width: "25ch" }}
+            />
+            <TextField
+              name="title"
+              value={formData.title}
+              onChange={handleChange}
+              label="Title"
+              id="filled-size-small"
+              variant="filled"
+              size="small"
+              sx={{ width: "25ch" }}
+            />
+            <TextField
+              name="phone"
+              value={formData.phone}
+              onChange={handleChange}
+              label="Phone"
+              id="filled-size-small"
+              variant="filled"
+              size="small"
+              sx={{ width: "25ch" }}
+            />
+            <Grid sx={{ display: isNotMobile ? "flex" : "block" }}>
```

Comment:
Maybe change this so that the "@hackbeanbot" part does not appear below the text field on mobile. 
Feel free to disregard.